doc/releases/changelog-dev.md

```diff
@@ -148,6 +148,16 @@
 
 <h3>Improvements</h3>
 
+* The function `qml.math.safe_squeeze` is introduced and `gradient_transform` allows
+  for QNode argument axes of size `1`.
+  [(#2080)](https://github.com/PennyLaneAI/pennylane/pull/2080)
+
+  `qml.math.safe_squeeze` wraps `qml.math.squeeze`, with slight modifications:
+  - When provided the `axis` keyword argument, axes that do not have size `1` will be
+    ignored, instead of raising an error.
+  - The keyword argument `exclude_axis` allows to explicitly exclude axes from the
+    squeezing.
```

Comment:
```suggestion
  `qml.math.safe_squeeze` wraps `qml.math.squeeze`, with slight modifications:

  - When provided the `axis` keyword argument, axes that do not have size `1` will be
    ignored, instead of raising an error.

  - The keyword argument `exclude_axis` allows to explicitly exclude axes from the
    squeezing.
```